react/states_and_effects/project_cv_application.md

```diff
@@ -81,3 +81,9 @@ Similar to Netlify, Vercel also offers a few ways to deploy, but we will be impo
 - [Cloudflare Pages Homepage](https://pages.cloudflare.com/)
 - [Cloudflare Pages documentation website](https://developers.cloudflare.com/pages)
 - [Vite deployment to Cloudflare Pages](https://developers.cloudflare.com/pages/framework-guides/deploy-a-vite3-project/) discusses these very steps
+
+### Additional resources
+
+This section contains helpful links to related content. It isn't required, so consider it supplemental.
+
+- If you want to include icons in your project, there is an alternative to inlining them as `<svg>`s or referencing them via an `<img>` tag. React icon component libraries such as [Lucide React](https://lucide.dev/guide/packages/lucide-react) wrap icons in components allowing you to reference them as an `<Icon />` component which provides for nicer developer experience. You can view all of their icons in their [icon catalog](https://lucide.dev/icons/).
```

Comment:
Had a further think and I'm not *quite* sure that this is particularly necessary, or at least if we do have it as an AR, whether the CV project is the most appropriate place to suggest it.

Icon component libraries aren't really all that different from general component libraries, just a subset of them. I would imagine that anyone who would really want to use icon libraries, or finds making custom components for icons etc. tedious etc., would be more than capable of finding that out themselves.

If we were to have the AR, to me it feels a little more suitable as an AR later in the course, like the lesson on styling.

@01zulfi any thoughts on this?

Reply:
This addition fits right into the [Component libraries section in the Styling React Applications lesson](https://www.theodinproject.com/lessons/node-path-react-new-styling-react-applications#component-libraries)

Reply:
Well I completely missed that section somehow xD That makes the decision much easier. I personally don't think icon component libraries are special enough to need a separate mention from what we already have there, so in my mind, we can just leave the lessons/projects as they are. Any differing thoughts?

Reply:
Most component libraries have their own Icon components so I guess we can mention that there are icon component only libraries. We can add this there as a lesson note or something, though whatever y'all decide

Reply:
In that case, probably the cleanest way is just to say something after the component library links like "There are also things like icon-only component libraries such as [Lucide-React](https://lucide.dev/guide/packages/lucide-react)."

@nikitarevenco You happy to make that change instead? i.e. Remove any changes to the CV project and add something simple like the above to the component library section of the Styling lesson? No need for anything more than a brief mention of the library and that's it.

Reply:
> In that case, probably the cleanest way is just to say something after the component library links like "There are also things like icon-only component libraries such as [Lucide-React](https://lucide.dev/guide/packages/lucide-react)."
> 
> @nikitarevenco You happy to make that change instead? i.e. Remove any changes to the CV project and add something simple like the above to the component library section of the Styling lesson? No need for anything more than a brief mention of the library and that's it.

So I added mention of icons there and did a slight overhaul of that section. Instead of putting everything into one place, I split up that section into 3 smaller sub-sections. let me know what you think

Reply:
I don't feel the section needs an overhaul at all, we only really want to just mention icon-only component libraries with an example. We don't really need to expand on the different kinds of component libraries and such, especially when we still recommend doing things more vanilla-y. If someone still wishes to use any component library, including just for icons, that's completely their choice. I would personally have all of them just under "component libraries" as one, and an individual can decide if they want to use whatever.

In the end, it's really not much work to use an img/svg tag or make your own icon component by that point (or later in the Node course). So since the icon library is purely a suggestion, I think we really just want something like the sentence I suggested in my previous comment added to the end.

Reply:
> I don't feel the section needs an overhaul at all, we only really want to just mention icon-only component libraries with an example. We don't really need to expand on the different kinds of component libraries and such, especially when we still recommend doing things more vanilla-y. If someone still wishes to use any component library, including just for icons, that's completely their choice. I would personally have all of them just under "component libraries" as one, and an individual can decide if they want to use whatever. i.e. the scope of that section is less "teach what different types of component libraries are and how you'd use them" and more just "introduce the idea of component libraries"
> 
> In the end, it's really not much work to use an img/svg tag or make your own icon component by that point (or later in the Node course). So since the icon library is purely a suggestion, I think we really just want something like the sentence I suggested in my previous comment added to the end.


Okay! I undid the overhaul.